English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_tens_comp_26_22_2.txt

```diff
@@ -0,0 +1 @@
+26 and 22 have the same number of tens. Compare the ones how many ones are  in number 26.
```

Comment:
Double space. 
Please get this reviewed in the application from me.

Reply:
Removed.